id/src/main/java/com/simprints/id/Application.kt

```diff
@@ -44,6 +45,11 @@ open class Application : CoreApplication(), Configuration.Provider {
         initApplication()
     }
 
+    override fun onTerminate() {
```

Comment:
Is this mainly for debug purposes?

Reply:
Added this to be 100% sure that the global scope is properly cancelled when the app is terminated.